pyproject.toml

```diff
@@ -102,7 +102,7 @@ types = [
     "mypy",
     "types-PyYAML",
     # Pinned because any higher requires urllib3>=2
```

Comment:
```suggestion
```

This fact is outdated and we can use newer Requests. @mr-cal WDYT about these two suggestions?